src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/8.sql

```diff
@@ -133,6 +133,14 @@ CREATE TYPE dbo.ResourceWriteClaimTableType_1 AS TABLE
     ClaimValue nvarchar(128) NOT NULL
 )
 
+CREATE TYPE dbo.BulkResourceWriteClaimTableType_1 AS TABLE
+(
+    ResourceTypeId smallint NOT NULL,
+    ResourceId varchar(64) COLLATE Latin1_General_100_CS_AS NOT NULL,
```

Comment:
I'd like for a surrogate id field (or an offset, which is what I think I called it in my branch) to be included in these so that the table types can be re-used for bulk upserting, even if we ignore the field when reindexing.

Reply:
Replaced `ResourceId` and `ResourceTypeId` with `Offset` in 46be3d9.